rest/data/project_test.go

```diff
@@ -329,6 +350,20 @@ func (s *ProjectConnectorGetSuite) TestGetProjectWithCommitQueueByOwnerRepoAndBr
 	s.NotNil(projRef)
 }
 
+func (s *ProjectConnectorGetSuite) TestGetProjectSettingsEvent() {
+	projRef := getProjectRef()
+	projectSettingsEvent, err := s.ctx.GetProjectSettingsEvent(projRef)
+	s.NoError(err)
+	s.NotNil(projectSettingsEvent)
+}
+
+func (s *ProjectConnectorGetSuite) TestGetProjectSettingsEventNoRepo() {
+	projRef := model.ProjectRef{}
```

Comment:
did you mean for this to be `getProjectRefNoRepo()`?

Reply:
yes